Makefile

```diff
@@ -21,12 +25,30 @@ test-install:
 .PHONY: test-install
 
 test-e2e:
-	sh openshift/e2e-tests-openshift.sh
+	sh openshift/e2e-tests.sh
 .PHONY: test-e2e
 
-test-origin-conformance:
-	sh TEST_ORIGIN_CONFORMANCE=true openshift/e2e-tests-openshift.sh
-.PHONY: test-origin-conformance
+# Requires ko 0.2.0 or newer.
+test-images:
+	for img in $(TEST_IMAGES); do \
+		ko resolve --tags=latest -RBf $$img ; \
```

Comment:
are we now cool, to use `ko` ? I am happy to merge this

Reply:
That's for local runs, not CI. As long as you have `ko` installed you're fine. And it makes it easier to build and push the image, IMO.